crates/task-impls/src/consensus/mod.rs

```diff
@@ -747,6 +752,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> ConsensusTaskState<TYPES, I>
                 let consensus = self.consensus.read().await;
                 consensus.metrics.number_of_timeouts.add(1);
             }
+            #[cfg(not(feature = "dependency-tasks"))]
```

Comment:
As mentioned in [Zulip](https://espresso.zulipchat.com/#narrow/stream/397846-Consensus-Team/topic/Proposal.20Task/near/436226938), looks like all the logic inside this `SendPayloadCommitmentAndMetadata` handling has been moved to the quorum proposal recv task, except for the `self.payload_commitment_and_metadata` setting. I think we should set it in the quorum proposal recv task as well, unless we find it's unnecessary, in which case we should remove it from here as well.

Reply:
It was unnecessary and it was removed. Thanks for highlighting!